molecule/provisioners/vagrantprovisioner.py

```diff
@@ -126,10 +128,10 @@ def default_platform(self):
                 0]['name']
 
         # default to first entry if no entry for platform exists or platform is false
-        if not self.m._state.get('default_platform'):
+        if not self.m._state.default_platform:
```

Comment:
I'm not sure I understand why this is even necessary.  Why cant we simply look at `self.m_args['--platform']`